Gopkg.toml

```diff
@@ -88,7 +88,7 @@ ignored = ["github.com/Sirupsen/logrus", "github.com/gravitational/planet/build/
 
 [[constraint]]
   name = "github.com/gravitational/satellite"
-  version = "=6.1.0"
+  branch = "sergei/bump_k8s_61x"
```

Comment:
Will be changed after merging https://github.com/gravitational/satellite/pull/120